src/System.Net.Http.WinHttpHandler/tests/FunctionalTests/System.Net.Http.WinHttpHandler.Functional.Tests.csproj

```diff
@@ -12,9 +12,6 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Windows_Debug|AnyCPU' " />
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Windows_Release|AnyCPU' " />
   <ItemGroup Condition=" '$(TargetsWindows)' == 'true' ">
-    <Compile Include="$(CommonTestPath)\System\Net\HttpTestServers.cs">
-      <Link>Common\System\Net\HttpTestServers.cs</Link>
-    </Compile>
     <Compile Include="$(CommonTestPath)\System\Net\HttpTestServers2.cs">
       <Link>Common\System\Net\HttpTestServers2.cs</Link>
```

Comment:
I will be doing that in the next PR today.  I didn't want to do it in this PR. I wanted the history to look clean.

 I'll also need to touch a bunch of other test projects as well when I do that.